build-tools/upload-junit-to-cloud.py

```diff
@@ -0,0 +1,65 @@
+from google.cloud import bigquery
+import xml.etree.ElementTree as ET
+import argparse
+
+# Uploading JUnit test results to BigQuery
+def upload(final: bool):
+    client = bigquery.Client()
+
+    dataset_id = 'test_results'
+    table_id = 'my_table'
```

Comment:
info: a little bit ugly name for table, but I just have left what was in an example, and it was too late to change because my report was ready :/